qtaf-core/src/test/java/de/qytera/qtaf/core/gson/serializer/StepAnnotationSerializerTest.java

```diff
@@ -0,0 +1,28 @@
+package de.qytera.qtaf.core.gson.serializer;
+
+import com.google.gson.JsonElement;
+import de.qytera.qtaf.core.guice.annotations.Step;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class StepAnnotationSerializerTest {
+    @Test(testName = "Test TestScenarioAnnotationSerializer")
+    public void testSerializer() throws NoSuchMethodException {
+        StepAnnotationSerializer serializer = new StepAnnotationSerializer();
+        Step step = DemoScenario.class.getMethod("demoStep").getAnnotation(Step.class);
+        JsonElement jsonElement = serializer.serialize(step, null, null);
+        Assert.assertEquals(jsonElement.toString(), "{\"name\":\"Demo Step\",\"description\":\"This is a demonstration step\"}");
```

Comment:
Please use text blocks for this to make it more readable:
```java

        Assert.assertEquals(jsonElement.toString(), """
                {"name":"Demo Step","description":"This is a demonstration step"}"""
        );
```

Reply:
ok, corrected